
Don't drop -Wall in the swig Perl bindings, otherwise building with
e.g. -Wformat-security might break.

https://bugzilla.redhat.com/show_bug.cgi?id=1037341

Index: subversion-1.14.2/subversion/bindings/swig/perl/native/Makefile.PL.in
===================================================================
--- subversion-1.14.2.orig/subversion/bindings/swig/perl/native/Makefile.PL.in
+++ subversion-1.14.2/subversion/bindings/swig/perl/native/Makefile.PL.in
@@ -61,7 +61,6 @@ chomp($cflags);
 # SWIG is using C++ style comments in an extern "C" code.
 $cflags =~ s/-ansi\s+//g;
 $cflags =~ s/-std=c89\s+//g;
-$cflags =~ s/-Wall//g;
 $cflags =~ s/-Wunused//g;
 $cflags =~ s/-Wshadow//g;
 $cflags =~ s/-Wstrict-prototypes//g;
